codalab/worker_manager/aws_batch_worker_manager.py

```diff
@@ -102,6 +102,8 @@ def start_worker_job(self):
             command.extend(['--exit-after-num-runs', str(self.args.worker_exit_after_num_runs)])
         if self.args.worker_exit_on_exception:
             command.extend(['--exit-on-exception'])
+        if self.args.worker_pass_down_termination:
+            command.extend(['--pass-down-termination'])
```

Comment:
I know this is orthogonal to this PR, but seems like this block can be pulled out and shared across all worker managers?

Reply:
Yeah, that's a good idea.